src/scales/timeScale.ts

```diff
@@ -47,6 +46,38 @@ export module Scales {
       var startTimeValue = endTimeValue - Plottable.MILLISECONDS_IN_ONE_DAY;
       return [new Date(startTimeValue), new Date(endTimeValue)];
     }
+
+    public scale(value: Date): number {
+      return this._d3Scale(value);
+    }
+
+    protected _getDomain() {
+      return this._d3Scale.domain();
+    }
+
+    protected _setActualDomain(values: Date[]) {
+      this._d3Scale.domain(values);
+    }
+
+    protected _getRange() {
+      return this._d3Scale.range();
+    }
+
+    protected _setRange(values: number[]) {
+      this._d3Scale.range(values);
+    }
+
+    public invert(value: number) {
+      return this._d3Scale.invert(value);
+    }
+
+    public getDefaultTicks(): Date[] {
+      return this._d3Scale.ticks(QuantitativeScale._DEFAULT_NUM_TICKS);
+    }
+
+    public _niceDomain(domain: Date[], count?: number) {
+      return Utils.D3Scale.niceDomain(<D3.Scale.QuantitativeScale> <D3.Scale.Scale> this._d3Scale, domain, count);
```

Comment:
Unless I'm mistaken `D3.Scale.TimeScale` is actually a child of `D3.Scale.QuantitativeScale` but the d3.d.ts file does not say this point.  Let me check/verify